akka-javasdk-maven/akka-javasdk-standalone-parent/pom.xml

```diff
@@ -0,0 +1,453 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.akka</groupId>
+        <artifactId>akka-javasdk-maven</artifactId>
```

Comment:
I guess we can move all common things to the grand-parent, but I wanted to make sure we agree on the approach of using a separate parent pom first.

Reply:
I was wondering if we could do it with a profile instead? Same parent pom, just a different config for the `docker-maven-plugin` .. then no changes would be needed to the app. One could decide to package for akka platform or for standalone via profile.

Reply:
That's a good idea. I have changed to a profile.